### DISCUSSION TITLE
Modelling of AWS VPCs in Netbox

### DISCUSSION THREAD

**Discussion by salman-uk** [netbox-community/netbox#14842] [General]
Hi,

I just wanted to get other user's opinions on what is the best way to model AWS VPCs in netbox.
I am currently adding the VPCs into Netbox as VRFs but not sure if this is the best way.

**Comment by markkuleinio** [comment]
I believe "best" is highly subjective. For me and my automation purposes this has worked fine for years:
- Site = VPC
- Site name = account id + VPC id
- Region = the cloud region
- Prefix = the VPC CIDR, in the site
